integration_tests/test_user_svc_client.js

```diff
@@ -185,6 +185,9 @@ async function getTokenForNewUser(userRequest, svcInfo) {
     });
   }
 
+  const identity = { identification: { token: newUserData.res.identification.token } };
+  await index.hwscUserSvc.verifyEmailToken(identity, svcInfo);
```

Comment:
Need to authenticate email before 
` const tokenData = await index.hwscUserSvc.getAuthToken(newUserRequest, svcInfo);` at line 195